app/src/lib/actions/neynar.ts

```diff
@@ -0,0 +1,18 @@
+import { getFarcasterUser } from "../neynar"
+
+export const getFarcasterUserData = async (fid: string) => {
+  try {
+    const user = await getFarcasterUser(fid)
+    if (user) {
+      return {
+        error: null,
+        user: user ?? null,
+      }
+    }
+  } catch (error: unknown) {
+    console.error("Error searching for user", (error as Error).message)
+    return {
+      error: "Error searching for user",
+    }
```

Comment:
this return error pattern works well when we want to inform user of an error. Fetching github data does not require user input and can / should fail silently. It's okay to send these error directly into dev console. External API calls should be able to fail without degrading user experience
tldr: let's remove the try / catch block